paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/Form.kt

```diff
@@ -47,16 +49,16 @@ import com.stripe.android.paymentsheet.specifications.FormItemSpec
 import com.stripe.android.paymentsheet.specifications.IdentifierSpec
 import com.stripe.android.paymentsheet.specifications.LayoutSpec
 import com.stripe.android.paymentsheet.specifications.ResourceRepository
+import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.flow.flatMapLatest
 import kotlinx.coroutines.flow.map
 import kotlinx.coroutines.launch
 import javax.inject.Inject
 import javax.inject.Named
 import javax.inject.Singleton
 
-internal val formElementPadding = 16.dp
-
 @ExperimentalAnimationApi
 @Composable
 internal fun Form(
```

Comment:
`fun` names should start with lowerCase, ditto to other functions in this file

Reply:
The convention in compose is to use upper case

Reply:
I would like to find a way to disable the warnings for these compose files/functions globally.  I will take a look and follow up in another PR.